.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
Why 15 instead of 16?

Reply:
I refer the existing workflow and it seems eslint-config supports 15.x. 
https://github.com/moneytree/eslint-config/blob/master/.github/workflows/node.js.yml

It seems good to add 16.x as well

Reply:
That file is old enough for 16 not to have existed :)
(in other words, I didn't maintain it 🙇)

But 15 has been superseded by 16 as "current", and is best left ignored and not used for anything IMHO. Major version info here: https://nodejs.org/en/about/releases/

Reply:
> But 15 has been superseded by 16 as "current", and is best left ignored and not used for anything IMHO. Major version info here: https://nodejs.org/en/about/releases/

Thanks for the info. I forgot that even-numbered release s will be LTS. I'm gonna fix it

Reply:
fixed